src/twisted/internet/_dumbwin32proc.py

```diff
@@ -178,13 +199,13 @@ def __init__(self, reactor, protocol, command, args, environment, path):
 
         if _PY3:
             # Make sure all the arguments are Unicode.
-            args = [_maybeMBCS(x) for x in args]
+            args = [_fsdecode(x) for x in args]
```

Comment:
This is not a new change in this PR, so not feedback for this PR, but all this transcoding stuff should happen _before_ any of the syscalls; if we're going to bail out due to an encoding error we should do it _fast_, before we do a bunch of potentially destructive work that I'm not entirely sure will even be cleaned up properly...